gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.1-bin.zip
```

Comment:
Did you use command for this upgrade or you just changed the version? Last time when I used the command for gradle upgrade, the command did changes to the `gradlew` scripts also.

Reply:
Yes I did, but I probably have made a mistake indeed

Reply:
fixed, thanks for spotting that one @violetagg !